Add sponsor page

keywordnew opened the pull request:
Also adding 2 venue sponsors from this year.

Older sponsors and new ones to follow in a different PR.

keywordnew committed:
Update footer content

keywordnew committed:
Add sponsor page

keywordnew committed:
Add link to sponsors

keywordnew committed:
Add another sponsor

Qard commented:
Should we maybe add Sauce and Clio too, for providing space for NodeSchool?

keywordnew commented:
> Should we maybe add Sauce and Clio too, for providing space for NodeSchool?

@Qard totally!

caffodian commented on src/pages/sponsors.js:
```diff
@@ -0,0 +1,57 @@
+
+import logoElysian from '../assets/images/elysian-logo.png'
+
+class Generic extends React.Component {
```
this coulllllld just be a function component, as it's not like this actually has any lifecycle hooks or anything. 
```
const Generic = (props) => ....allTheStuffInRender
```

but I might be being silly

caffodian replied:
(and you can even remove the lodash dependency by doing so, I think)

caffodian replied:
also should maybe rename the component to SponsorPage or something.  In case someone ever opens it up in React devtools.

caffodian replied:
Now I feel very silly for providing real feedback, I've been actually working today, give me a break LOL

keywordnew replied:
Real feedback is super appreciated! It's open source! We must have code standards 😄

keywordnew replied:
Converted @caffodian 🥳

caffodian reviewed (approved):
i'm just approving because I can't tell how seriously my feedback should be taken

caffodian commented on src/pages/sponsors.js:
```diff
@@ -0,0 +1,57 @@
+                </p>
+              </li>
+            </ul>
+          </section>
```
on a related note we totally could make a `sponsors.json` file or something (or if you wanted to go overkill, one of the gatsby data sources).  It could store an array of sponsors with description, url, and name, and they would just get rendered here in random order.

THIS IS PROBABLY OVERKILL FOR A SPONSOR PAGE OF 2, DON'T TAKE THIS SERIOUSLY

Qard replied:
Why even _write_ code if you're not going to over-engineer it? :trollface:

caffodian replied:
this is true.  Also, within the review process, we've updated the number of sponsors to _4_

keywordnew replied:
Did I mention how much fun I have working with you both on this?! 💥🖤💥

keywordnew replied:
To get this merged, would it make sense to make a new issue for this?

Is it a candidate for a good-first-issue?

Qard replied:
Yeah, definitely cool landing without this change. Might be a good candidate for a NodeSchool Open Source Day PR. ;)

keywordnew committed:
Add sponsors

Related comment:
https://github.com/cambiecollective/cambiecollective.github.io/pull/28#issuecomment-453321616

keywordnew committed:
Add another link to sponsors

keywordnew committed:
Rename component

keywordnew committed:
Convert to stateless functional component

keywordnew commented:
Updated with new commits! 

I'm really sorry it took this much time to address the review comments. I'll do better at moving things along. Can't believe it took a cancelled dental appointment to get this rolling :D

PTAL?

Qard commented on src/pages/sponsors.js:
```diff
@@ -0,0 +1,75 @@
+                offices in Downtown Vancouver.
+                Their crew is welcoming, 
+                their platform covers a lot of testing needs, 
+                and their sauce is covers low and high ends of the Scoville scale.
```
`s/sauce is/sauces/`?

Qard commented on src/pages/sponsors.js:
```diff
@@ -0,0 +1,75 @@
+              <h3><a href='https://7gate.vc'>💻 7 Gate Ventures</a></h3>
+              <p>
+                7 Gate Ventures have shared their spaces with several of our meetups.
+                Their crew is welcoming, 
```
Might be nice to have a little more linguistic variety than just repeating this exact phrase four times. 😅

keywordnew replied:
The goal is to emphasize that point :) Alternatives?

keywordnew committed:
Fix typo

keywordnew committed:
Fix icons for 2 new meetups

keywordnew committed:
Reduce repetitiveness in copy

keywordnew committed:
Update Nav

keywordnew committed:
Add incubating meetup
